28: Traceback (most recent call last):
28:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
28:     from cagnet.nn.conv import GCNConv
28:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
28:     from .gcn_conv import GCNConv
28:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
28:     """commenting out spmm call
28:        ^
28: SyntaxError: invalid syntax
29: Traceback (most recent call last):
29:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
29:     from cagnet.nn.conv import GCNConv
29:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
29:     from .gcn_conv import GCNConv
29:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
29:     """commenting out spmm call
29:        ^
29: SyntaxError: invalid syntax
31: Traceback (most recent call last):
31:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
31:     from cagnet.nn.conv import GCNConv
31:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
31:     from .gcn_conv import GCNConv
31:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
31:     """commenting out spmm call
31:        ^
31: SyntaxError: invalid syntax
30: Traceback (most recent call last):
30:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
30:     from cagnet.nn.conv import GCNConv
30:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
30:     from .gcn_conv import GCNConv
30:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
30:     """commenting out spmm call
30:        ^
30: SyntaxError: invalid syntax
 7: Traceback (most recent call last):
 7:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
 7:     from cagnet.nn.conv import GCNConv
 7:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
 7:     from .gcn_conv import GCNConv
 7:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
 7:     """commenting out spmm call
 7:        ^
 7: SyntaxError: invalid syntax
 4: Traceback (most recent call last):
 4:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
 4:     from cagnet.nn.conv import GCNConv
 4:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
 4:     from .gcn_conv import GCNConv
 4:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
 4:     """commenting out spmm call
 4:        ^
 4: SyntaxError: invalid syntax
 5: Traceback (most recent call last):
 5:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
 5:     from cagnet.nn.conv import GCNConv
 5:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
 5:     from .gcn_conv import GCNConv
 5:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
 5:     """commenting out spmm call
 5:        ^
 5: SyntaxError: invalid syntax
 6: Traceback (most recent call last):
 6:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
 6:     from cagnet.nn.conv import GCNConv
 6:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
 6:     from .gcn_conv import GCNConv
 6:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
 6:     """commenting out spmm call
 6:        ^
 6: SyntaxError: invalid syntax
 2: Traceback (most recent call last):
 2:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
 2:     from cagnet.nn.conv import GCNConv
 2:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
 2:     from .gcn_conv import GCNConv
 2:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
 2:     """commenting out spmm call
 2:        ^
 2: SyntaxError: invalid syntax
 0: Traceback (most recent call last):
 0:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
 0:     from cagnet.nn.conv import GCNConv
 0:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
 0:     from .gcn_conv import GCNConv
 0:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
 0:     """commenting out spmm call
 0:        ^
 0: SyntaxError: invalid syntax
 3: Traceback (most recent call last):
 3:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
 3:     from cagnet.nn.conv import GCNConv
 3:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
 3:     from .gcn_conv import GCNConv
 3:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
 3:     """commenting out spmm call
 3:        ^
 3: SyntaxError: invalid syntax
 1: Traceback (most recent call last):
 1:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
 1:     from cagnet.nn.conv import GCNConv
 1:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
 1:     from .gcn_conv import GCNConv
 1:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
 1:     """commenting out spmm call
 1:        ^
 1: SyntaxError: invalid syntax
20: Traceback (most recent call last):
20:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
20:     from cagnet.nn.conv import GCNConv
20:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
20:     from .gcn_conv import GCNConv
20:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
20:     """commenting out spmm call
20:        ^
20: SyntaxError: invalid syntax
56: Traceback (most recent call last):
56:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
57: Traceback (most recent call last):
57:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
58: Traceback (most recent call last):
58:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
59: Traceback (most recent call last):
59:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
56:     from cagnet.nn.conv import GCNConv
56:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
57:     from cagnet.nn.conv import GCNConv
57:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
58:     from cagnet.nn.conv import GCNConv
58:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
59:     from cagnet.nn.conv import GCNConv
59:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
56:     from .gcn_conv import GCNConv
56:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
56:     """commenting out spmm call
56:        ^
56: SyntaxError: invalid syntax
57:     from .gcn_conv import GCNConv
57:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
57:     """commenting out spmm call
57:        ^
57: SyntaxError: invalid syntax
58:     from .gcn_conv import GCNConv
58:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
58:     """commenting out spmm call
58:        ^
58: SyntaxError: invalid syntax
59:     from .gcn_conv import GCNConv
59:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
59:     """commenting out spmm call
59:        ^
59: SyntaxError: invalid syntax
23: Traceback (most recent call last):
23:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
23:     from cagnet.nn.conv import GCNConv
23:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
23:     from .gcn_conv import GCNConv
23:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
23:     """commenting out spmm call
23:        ^
23: SyntaxError: invalid syntax
60: Traceback (most recent call last):
60:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
61: Traceback (most recent call last):
61:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
62: Traceback (most recent call last):
62:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
63: Traceback (most recent call last):
63:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
62:     from cagnet.nn.conv import GCNConv
62:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
63:     from cagnet.nn.conv import GCNConv
63:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
61:     from cagnet.nn.conv import GCNConv
61:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
60:     from cagnet.nn.conv import GCNConv
60:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
60:     from .gcn_conv import GCNConv
60:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
60:     """commenting out spmm call
60:        ^
60: SyntaxError: invalid syntax
61:     from .gcn_conv import GCNConv
61:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
61:     """commenting out spmm call
61:        ^
61: SyntaxError: invalid syntax
62:     from .gcn_conv import GCNConv
62:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
62:     """commenting out spmm call
62:        ^
62: SyntaxError: invalid syntax
63:     from .gcn_conv import GCNConv
63:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
63:     """commenting out spmm call
63:        ^
63: SyntaxError: invalid syntax
22: Traceback (most recent call last):
22:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
22:     from cagnet.nn.conv import GCNConv
22:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
22:     from .gcn_conv import GCNConv
22:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
22:     """commenting out spmm call
22:        ^
22: SyntaxError: invalid syntax
21: Traceback (most recent call last):
21:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
21:     from cagnet.nn.conv import GCNConv
21:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
21:     from .gcn_conv import GCNConv
21:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
21:     """commenting out spmm call
21:        ^
21: SyntaxError: invalid syntax
16: Traceback (most recent call last):
16:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
16:     from cagnet.nn.conv import GCNConv
16:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
16:     from .gcn_conv import GCNConv
16:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
16:     """commenting out spmm call
16:        ^
16: SyntaxError: invalid syntax
18: Traceback (most recent call last):
18:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
18:     from cagnet.nn.conv import GCNConv
18:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
18:     from .gcn_conv import GCNConv
18:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
18:     """commenting out spmm call
18:        ^
18: SyntaxError: invalid syntax
17: Traceback (most recent call last):
17:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
17:     from cagnet.nn.conv import GCNConv
17:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
17:     from .gcn_conv import GCNConv
17:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
17:     """commenting out spmm call
17:        ^
17: SyntaxError: invalid syntax
19: Traceback (most recent call last):
19:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
19:     from cagnet.nn.conv import GCNConv
19:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
19:     from .gcn_conv import GCNConv
19:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
19:     """commenting out spmm call
19:        ^
19: SyntaxError: invalid syntax
37: Traceback (most recent call last):
37:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
37:     from cagnet.nn.conv import GCNConv
37:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
37:     from .gcn_conv import GCNConv
37:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
37:     """commenting out spmm call
37:        ^
37: SyntaxError: invalid syntax
38: Traceback (most recent call last):
38:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
38:     from cagnet.nn.conv import GCNConv
38:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
38:     from .gcn_conv import GCNConv
38:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
38:     """commenting out spmm call
38:        ^
38: SyntaxError: invalid syntax
36: Traceback (most recent call last):
36:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
36:     from cagnet.nn.conv import GCNConv
36:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
36:     from .gcn_conv import GCNConv
36:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
36:     """commenting out spmm call
36:        ^
36: SyntaxError: invalid syntax
39: Traceback (most recent call last):
39:   File "/global/u2/j/jinimukh/CAGNET/examples/gcn_1d.py", line 20, in <module>
39:     from cagnet.nn.conv import GCNConv
39:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/__init__.py", line 1, in <module>
39:     from .gcn_conv import GCNConv
39:   File "/global/homes/j/jinimukh/CAGNET/cagnet/nn/conv/gcn_conv.py", line 118
39:     """commenting out spmm call
39:        ^
39: SyntaxError: invalid syntax
srun: error: nid008485: tasks 28,30-31: Exited with exit code 1
srun: launch/slurm: _step_signal: Terminating StepId=6053236.3
 0: slurmstepd: error: *** STEP 6053236.3 ON nid008472 CANCELLED AT 2023-03-14T06:44:38 ***
srun: error: nid008473: tasks 4,6-7: Exited with exit code 1
srun: error: nid008472: tasks 0,2-3: Exited with exit code 1
srun: error: nid008500: tasks 56,59: Terminated
srun: error: nid008501: tasks 60,62: Terminated
srun: error: nid008480: tasks 16,18-19: Terminated
srun: error: nid008481: tasks 20-21,23: Terminated
srun: error: nid008492: tasks 40-41,43: Terminated
srun: error: nid008477: tasks 12-13,15: Terminated
srun: error: nid008484: tasks 25-27: Terminated
srun: error: nid008500: task 57: Terminated
srun: error: nid008488: task 34: Terminated
srun: error: nid008476: tasks 9,11: Terminated
srun: error: nid008489: tasks 36,38-39: Terminated
srun: error: nid008501: task 61: Terminated
srun: error: nid008497: tasks 53-54: Terminated
srun: error: nid008493: tasks 44,46: Terminated
srun: error: nid008496: tasks 48,50: Terminated
srun: error: nid008476: task 8: Terminated
srun: error: nid008493: task 47: Terminated
srun: error: nid008488: tasks 32,35: Terminated
srun: error: nid008497: task 55: Terminated
srun: error: nid008485: task 29: Exited with exit code 1
srun: error: nid008473: task 5: Exited with exit code 1
srun: error: nid008472: task 1: Exited with exit code 1
srun: error: nid008500: task 58: Terminated
srun: error: nid008496: task 51: Terminated
srun: error: nid008501: task 63: Terminated
srun: error: nid008492: task 42: Terminated
srun: error: nid008493: task 45: Terminated
srun: error: nid008489: task 37: Terminated
srun: error: nid008497: task 52: Terminated
srun: error: nid008476: task 10: Terminated
srun: error: nid008477: task 14: Terminated
srun: error: nid008480: task 17: Terminated
srun: error: nid008481: task 22: Terminated
srun: error: nid008488: task 33: Terminated
srun: error: nid008484: task 24: Terminated
srun: error: nid008496: task 49: Terminated
srun: Force Terminated StepId=6053236.3
